healsparse/__init__.py

```diff
@@ -24,3 +24,4 @@
 from .geom import Circle, Ellipse, Polygon, realize_geom
 from .utils import WIDE_MASK
 from .cat_healsparse_files import cat_healsparse_files
+from .bitSparseMap import BitSparseMap
```

Comment:
This is an internal construct and so I don't think should be part of the public API.